beitragsordnung.md

```diff
@@ -18,22 +21,49 @@ Die Mitgliederversammlung des Vereins Welcome Werkstatt e.V. hat am 9.11.2019 fo
+5. Zahlungsweise
+5.1 Bei Nutzung des Lastschriftverfahrens und Unregelmäßigkeiten auf den Konten des Vereinsmitglieds – z.B. mangelnde Deckung, Änderung der Kontonummer, der Anschrift, des Namens u.a. ohne Information an den Verein – werden pro Mahnung etwaige Rücklastschriftgebühren und eine Mahngebühr von 2,50 Euro erhoben.
 
-7. Der Verein erhebt eine Aufnahmegebühr von 10 Euro, die nach Aufnahme in den Verein fällig wird und mit dem ersten Mitgliedsbeitrag zusammen überwiesen werden kann oder gemeinsam mit dem ersten Mitgliedsbeitrag im Lastschriftverfahren eingezogen wird.
+5.2 Der Mitgliedsbeitrag kann auch per Überweisung entrichtet werden. Barzahlung des Mitgliedsbeitrags ist ausgeschlossen. 
 
-8. Bei Nutzung des Lastschriftverfahrens und Unregelmäßigkeiten auf den Konten des Vereinsmitglieds – z.B. mangelnde Deckung, Änderung der Kontonummer, der Anschrift, des Namens u.a. ohne Information an den Verein – werden pro Mahnung etwaige Rücklastschriftgebühren und eine Mahngebühr von 2,50 Euro erhoben.
+5.3. Es können Umlagen und / oder Sachleistungen von den Mitgliedern erhoben werden. Die Erhebung von Umlagen und / oder Sachleistungen muss von der Mitgliederversammlung beschlossen werden.
 
-9. Es können Umlagen und / oder Sachleistungen von den Mitgliedern erhoben werden. Die Erhebung von Umlagen und / oder Sachleistungen muss von der Mitgliederversammlung beschlossen werden.
 
-10. Alle ordentlichen Mitglieder mit Stimmrecht ab dem vollendeten 16. Lebensjahr und bis zu vollendeten 60. Lebensjahr müssen jährlich 3 Stunden Arbeit zum Erhalt und / oder zur Pflege der vereinseigenen Einrichtungen und Anlagen erbringen. Wird die Anzahl der Arbeitsstunden nicht erfüllt, erhebt der Verein pro nicht geleistete Stunde 3 Euro. Der eventuell fällig werdende Betrag wird per Lastschrifteinzug im Monat abgebucht, der auf den Monat folgt, in dem das Mitglied über die Abrechnung der Stunden informiert wurde, oder wird durch das Mitglied in bar oder per Überweisung beglichen.
+6. Sonstiges
+6.1. Anschriften-, Namens- und Kontenänderungen sowie Veränderungen, die zu einer anderen Bemessung der Beitragshöhe führen, sind dem Vorstand unverzüglich mitzuteilen.
 
-11. Anschriften-, Namens- und Kontenänderungen sowie Veränderungen, die zu einer anderen Bemessung der Beitragshöhe führen, sind dem Vorstand unverzüglich mitzuteilen.
+6.2. Wird der Beitrag trotz schriftlicher Mahnung nicht fristgemäß entrichtet, führt dies gemäß Satzung § 4 nach einem Verzug mit drei Monatsbeiträgen zum Ausschluss des Mitgliedes. Die Verbindlichkeiten bleiben davon unberührt. Für einfache Mahnschreiben ist der Verein berechtigt, ohne Nachweisführung einen Betrag von 2,50 Euro zu berechnen.
```

Comment:
Eine Voraussetzung für den Ausschluss eines Mitglieds kann m.E. nicht in der Beitragsordnung geregelt werden - das müsste in die Satzung.

Reply:
Es steht bereits unter § 4 was in der Satzung. Die Beitragsordnung konkretisiert hier scheinbar. In §4 der Satzung steht "Die Mitgliedschaft endet: 4. bei Mitgliedern, die sich mit mehr als drei Monatsbeiträgen im Verzug befinden, automatisch."

Reply:
Ah, das hatte ich übersehen, dass das schon so in der Satzung geregelt ist. Dann finde ich es gut so 👍